runtime/lib/backend/openqasm/OpenQasmBuilder.hpp

```diff
@@ -33,7 +33,8 @@ namespace Catalyst::Runtime::Device::OpenQasm {
  */
 enum class BuilderType : uint8_t {
     Common, // = 0
-    Braket,
+    BraketRemove,
```

Comment:
Why `BraketRemove`?

Reply:
Supposed to be `BraketRemote` :)

Reply:
Fixed [here](https://github.com/PennyLaneAI/catalyst/blob/746b903dd9a8a899124de42067a255f2c46b4fe8/runtime/lib/backend/openqasm/OpenQasmBuilder.hpp#L36)!